pkgs/top-level/python-packages.nix

```diff
@@ -8036,6 +8036,8 @@ self: super: with self; {
 
   pylatexenc = callPackage ../development/python-modules/pylatexenc { };
 
+  pylatex = callPackage ../development/python-modules/pylatex { };
```

Comment:
It might make sense to ensure these are properly ordered alphabetically as well.

Reply:
You're right!
I rebased with the master branch and made sure the packages are ordered correctly!